core/src/main/java/com/yahoo/ycsb/workloads/CoreWorkload.java

```diff
@@ -505,7 +525,7 @@ public String buildKeyName(long keynum) {
     if (!orderedinserts) {
       keynum = Utils.hash(keynum);
     }
-    return "user" + keynum;
+	return "user"+String.format("%0"+zeropadding+"d",keynum);
```

Comment:
Not sure it makes sense but can we short circuit if there is no padding with an if statement? That would avoid a slowdown compared to the original code?